### PR TITLE
Fix more select_console waits

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: tests/qam-kgraft/update_kgraft.pm [hunk 1]
```diff
@@ -153,7 +153,7 @@ sub post_fail_hook() {
     type_string("logout\n");
 
     # reconnect to svirt console
-    my $svirt = select_console('svirt');
+    my $svirt = select_console('svirt', await_console => 0);
```
Comment (coolo) [inline tests/qam-kgraft/update_kgraft.pm]: again, I think it doesn't make sense that you have to do this for every svirt, if you can just as well fix it in the function

Comment (okurz) [inline tests/qam-kgraft/update_kgraft.pm]: if it makes sense in all cases, sure

**Comment by okurz** [comment]
updated, for now only a simple fix. more will follow

File: tests/qam-kgraft/update_kgraft.pm (hunk 1 above)
Comment (okurz) [inline tests/qam-kgraft/update_kgraft.pm]: done, please recheck

**Comment by okurz** [comment]
@richiejp @coolo

File: tests/autoyast/installation.pm [hunk 2]
```diff
@@ -36,8 +36,6 @@ sub save_logs_and_continue {
     my $name = shift;
     # save logs and continue
     select_console 'install-shell';
-    sleep 5;
-    wait_idle(5);
     assert_screen ["inst-console"];
```
Comment (coolo) [inline tests/autoyast/installation.pm]: any idea why we're passing this as array?

Comment (coolo) [inline tests/autoyast/installation.pm]: and why you leave it in here, but remove it in others

Comment (okurz) [inline tests/autoyast/installation.pm]: cause I'm human -> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2593

**Pull request merged**